Sources/DarkModeCore/UIImage+DarkModeKit.m

```diff
@@ -11,8 +11,26 @@
 
 @implementation UIImage (DarkModeKit)
 
-+ (void)load {
-  [UIImage dm_swizzleInstanceMethod:@selector(isEqual:) to:@selector(dm_isEqual:)];
++ (void)dm_swizzleIsEqual {
+  static dispatch_once_t onceToken;
+  dispatch_once(&onceToken, ^{
+    SEL selector = @selector(isEqual:);
+    [self dm_swizzleSelector:@selector(isEqual:) withBlock:^id _Nonnull(IMPContainer *container) {
+      return ^BOOL (UIImage *self, UIImage *other) {
+        /// On iOS 13, UIImage `isEqual:` somehow changes internally and doesn't work for `NSProxy`,
+        /// here we forward the message to internal images manually
+        UIImage *realSelf = self;
+        UIImage *realOther = other;
+        if (object_getClass(self) == DMDynamicImageProxy.class) {
+          realSelf = ((DMDynamicImageProxy *)self).resolvedImage;
+        }
+        if (object_getClass(other) == DMDynamicImageProxy.class) {
+          realOther = ((DMDynamicImageProxy *)other).resolvedImage;
+        }
+        return ((BOOL (*)(UIImage *, SEL, UIImage *))container.imp)(realSelf, selector, realOther);
+      };
+    }];
+  });
```

Comment:
@icodesign used `load` in here because we didn't call `setup` in share extension, and lead to crash. keep in mind when we upgrade DarkModeKit in our codebase 🙈

Reply:
Can `initialize` work?